docs/source/en/model_doc/chameleon.md

```diff
@@ -54,8 +55,15 @@ The original code can be found [here](https://github.com/facebookresearch/chamel
 
 - Chameleon generates in chat format which means that the generated text will always be the "assistant's turn". You can enable a text completion generation by passing `return_for_text_completion=True` when calling the processor.
 
-> [!NOTE]
-> Chameleon implementation in Transformers uses a special image token to indicate where to merge image embeddings. For special image token we didn't add a new one but used one of the reserved tokens: `<reserved08707>`. You have to add `<image>` to your prompt in the place where the image should be embedded for correct generation.
+## How are images handled by Chameleon in Transformers?
+
+The `ChameleonProcessor` takes as input strings and images and preprocesses them. The strings are tokenized as you would see with other models with one exception, for every instance of the substring `<image>` in the input strings you will get a sequence of 1026 tokens: `<img_start>`, 1024 * `<reserved08707>`, `<img_end>`. This `<reserved08707>` token indicates where to merge the image tokens later when the `ChameleonModel` is called. All these tokens, those for the text with the interleaved image sentinel and placeholder tokens are all placed in the output dictionary mapped to the key `input_ids`.
+
+The images passed to `ChameleonProcessor` are processed through a sequence of transformations: format conversion, center cropping, resizing, normalizing etc. The processed images are put into the output dictionary as a tensor mapped to the key `pixel_values`.
+
+The conversion of the `pixel_values` tensor into image tokens and substitution into the placeholder tokens in `input_ids` happens when you call the `ChameleonModel`. This conversion is done in two steps, the first of which uses a [Vector Quantized Variational AutoEncoder (VQ-GAE)](https://arxiv.org/abs/1711.00937) to generate image tokens which are then mapped to Byte Pair Encoding (BPE) tokens. These image tokens are then swapped into the placeholder tokens in `input_ids`.
+
+Tip: If you want to see the image tokens a given image maps to, you can use the `ChameleonModel`'s `get_image_tokens` method, passing it the `pixel_values`.
```

Comment:
Seems a bit too detailed, imo. Maybe we can wrap this in a "Note" section saying that image tokenization happens when calling the model and if one wants to obtain image tokens, then one should use `get_image_tokens()` method

Since the definition of what sis input ids and pixel values is already in model's docstring, this seems redundant.